src/main/webapp/feedback.jsp

```diff
@@ -0,0 +1,87 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%
+  boolean canOpen = (boolean) request.getAttribute("feedbackOpen");
+  String role = (String) request.getAttribute("role");
+  pageContext.setAttribute("feedbackOpen", canOpen);
+  pageContext.setAttribute("role", role); 
+%>
+
+<c:choose>
+  <c:when test= "${canOpen == false}">
+    <h2 style="text-align: center">You may not submit feedback yet.</h2>
+  </c:when>
+  <c:otherwise>
+    <c:choose>
+      <c:when test= "${role == 'Interviewee'}">
+        <h1 class="text-center">Feedback</h1>
+        <h4 class="text-center">Please submit your feedback for your interviewer below.</h4>
+        <h5 class="text-center lead">Please enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree).</h5>
+        <!-- TODO: Send email on submit -->
+        <form>
+          <div class="form-group">
+            <div class="form-group form-inline"> 
+              <label for="question1">I was comfortable during the interview:</label><br>
+              <input type="number" class="form-control" min="1" max="10" id="question1" name="question1"><br>
```

Comment:
This might look better if these boxes are right-aligned.

(I don't feel strongly about this)

Reply:
Done